content/docs/atom-guide/dev-guide/shaders/azsl/shader-variant-options.md

```diff
@@ -1,10 +1,46 @@
 ---
-linktitle: Shader Variant Options
-title: Design of Shader Variant Options
-description: Learn about the design principles and purpose of AZSL Shader Variant Options in the Atom Renderer. 
-toc: true
-weight: 300
-draft: true
+linktitle: Shader variant options
+title: Shader variant options in Amazon Shading Language (AZSL)
```

Comment:
```suggestion
linktitle: Shader Variant Options
title: Shader Variant Options in Amazon Shading Language (AZSL)
```
* Use camel casing for titles. For example: Shader **V**ariant **O**ptions

Reply:
Fixed.